cves/CVE-2017-9996.yml

```diff
@@ -240,8 +260,10 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: |
+    This defect exists as part of the avcodec a utility that encodes and
+    decodes various different media formats.
```

Comment:
Might want to mention the file type and action that is being done within this file (.cdxl and encoding/decoding)